bynder_sdk/util.py

```diff
@@ -6,7 +6,9 @@
 
 
 def api_endpoint_url(session, endpoint):
-    return 'https://{}/api{}'.format(session.bynder_domain, endpoint)
+    if endpoint.startswith('/v4'):
+        return 'https://{}/api{}'.format(session.bynder_domain, endpoint)
```

Comment:
Not all "old" routes start with `v4` though. The pybynder endpoints are just `/api/something`.

So maybe instead check if it starts with /v6 or /v7. Or just add "/api" in front of the old endpoints so we don't have to do this check here and it is more explicit to the reader of the code.